lambda.tf

```diff
@@ -0,0 +1,49 @@
+# ------------------------------------------------------------------------------
+# The AWS Lambda function that is used to disable the access of
+# inactive IAM users.  The ZIP file is created with:
+# http://github.com/cisagov/disable-inactive-iam-users-lambda
+# ------------------------------------------------------------------------------
+
+resource "aws_lambda_function" "disable_inactive_iam_users" {
+  description      = var.lambda_function_description
+  filename         = var.lambda_zip_filename
+  function_name    = var.lambda_function_name
+  handler          = "lambda_handler.handler"
+  memory_size      = 128
+  role             = aws_iam_role.lambdaexecution_role.arn
+  runtime          = "python3.9"
```

Comment:
Is there some reason why we wouldn't want to update this to run with `python3.11` or `python3.12`?

https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
In this case, I'm fine with getting ahead of the skeleton.  Let us know what you decide and I'll resolve this conversation either way.

Reply:
Our Terraform provider constraints only allow us to support Python 3.10 or earlier, so I will go with that.

Reply:
Please see commit 3ddb152ad5f212d25fae4204647e4327a78df46b and commit cisagov/disable-inactive-iam-users-lambda@311f8b8d2ea7997042970e4b1d025173f49ac28a.

Reply:
Also, we need to move to a Terraform provider that supports Python 3.12 (required for what we are building over in https://github.com/cisagov/cyhy-kevsync-lambda/tree/first-commits-felddy).